### PR TITLE
OADP-3184 cherrypicked to OCP 4.12

### PR DESCRIPTION
**Pull request by sbeskin-redhat**
Resolves: https://issues.redhat.com/browse/OADP-3184
OADP-1.3
OCP-4.12

### CODE REVIEW

File: backup_and_restore/application_backup_and_restore/backing_up_and_restoring/backing-up-applications.adoc [hunk 1]
```diff
@@ -40,6 +39,7 @@ You can schedule backups by creating a `Schedule` CR instead of a `Backup` CR. S
 
 This issue has been resolved in the OADP 1.1.6 and OADP 1.2.2 releases, therefore it is recommended that users upgrade to these releases.
 
+For more information, see xref:../../../backup_and_restore/application_backup_and_restore/troubleshooting.adoc#oadp-restic-restore-failing-psa-policy_oadp-troubleshooting[Restic restore partially failing on OCP 4.14 due to changed PSA policy].
```
Comment (aireilly) [inline backup_and_restore/application_backup_and_restore/backing_up_and_restoring/backing-up-applications.adoc:42]: Can you double check this xref. I think it is not correct.

Comment (sbeskin-redhat) [reply backup_and_restore/application_backup_and_restore/backing_up_and_restoring/backing-up-applications.adoc]: @aireilly 
Yes, that xref was indeed broken, thank you! I replaced it with a link and ran the check again. The check failed two times and then passed. I suspect that the previous failures weren't caused by the broken link but rather by some flakiness of the checks.
Thank you for your help!

**Pull request merged**